app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="add_shortcut_description">Wil je dat de kleur van Jockey\'s snelkoppling-icoon hetzelfde is als je huidige thema? Jockey zal herstarten en alle icoontjes zullen verwijderd worden.</string>
+    <string name="uses_aosp_header">Android Support Libraries</string>
+    <string name="uses_header">Jockey gebruikt de volgende open-source onderdelen:</string>
+    <string name="uses_aosp_detail">Support-v4 27.1.0, Appcompat-v7 27.1.0, RecyclerView 27.1.0, CardView 27.1.0, Design 27.1.0, Palette 27.1.0, Preference-v14 27.1.0, Room 1.1.0; gedistributeerd door het Android Open Source Project onder de Apache License, versie 2.0</string>
+    <string name="uses_exoplayer_detail">Versie 2.6.1; gedistributeerd door Google Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_libflac_detail">Version 1.3.1; gedistributeerd door Xiph.Org Foundation onder een BSD licentie</string>
+    <string name="uses_dagger_detail">Versie 2.6; gedistributeerd door Google Inc. en Square Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_java_detail">Versie 1.6.1; gedistributeerd door Netflix, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_android_detail">Versie 1.2.1; gedistributeerd door RxAndroid onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_permissions_detail">Versie 0.7.0; gedistributeerd door Thomas Bruyelle (tbruyelle) onder een Apache License, versie 2.0</string>
+    <string name="uses_retrofit_detail">Versie 2.0.2; gedistributeerd door Square, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_gson_detail">Versie 2.7; gedistributeerd door Google, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_glide_detail">Versie 3.7.0; gedistributeerd door Google, Inc. onder een BSD licensie</string>
+    <string name="uses_vert_slider_detail">Versie 0.5.2; gedistributeerd door Haruki Hasegawa onder een Apache License, versie 2.0</string>
+    <string name="uses_fastscroll_detail">Versie 1.0.16; gedistributeerd door Tim Malseed onder een Apache License, versie 2.0</string>
+    <string name="uses_seek_arc_detail">Versie 1.1; gedistributeerd door Triggertrap Ltd (Neil Davies) onder een MIT License</string>
+    <string name="uses_timber_detail">Versie 1.1; gedistributeerd door Triggertrap Ltd (Neil Davies) onder een MIT License</string>
+    <string name="about_description">Jockey is een muziekspeler voor Android 4.1 en hoger, gefocused op simpliciteit, snelheid en design gedistributeerd onder een Apache 2.0 licentie.</string>
+    <string name="empty_included_dirs_detail">Jockey kan beperkt worden om in alleen specifieke mappen te doorzoeken, druk op de “+” knop om een map in te sluiten.</string>
+    <string name="empty_excluded_dirs_detail">Jockey kan beperkt worden om sommige mappen over te slaan, druk op de “+” knop om een map uit te sluiten.</string>
```

Comment:
Looks like the quotes here still need to get converted to the HTML codes.

Reply:
Ah thanks, I use the Translations Editor in Android Studio and it shows `"+"` instead of `&#8220;+&#8221;` so I was unaware of this. (same thing with `"%1$s"`)